app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
```

Comment:
This controller needs to be in the `content_manegment` namspace

Reply:
updated